information-schema/information-schema-placement-policies.md

```diff
@@ -0,0 +1,72 @@
+---
+title: PLACEMENT_POLICIES
+summary: 了解 information_schema 表 `PLACEMENT_POLICIES`。
+---
```

Comment:
```suggestion
aliases: ['/zh/tidb/dev/information-schema-placement-rules']
---
```

Reply:
What is this? And how should it be in the english version #7885?

Reply:
Aliases are for redirects. When a user visits the deleted page, the server will redirect them to the new page rather than return 404. 
I'll add aliases for the English version, too.